src/test/java/com/epam/test_generator/controllers/LoginControllerTest.java

```diff
@@ -0,0 +1,110 @@
+package com.epam.test_generator.controllers;
+
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.epam.test_generator.dto.LoginUserDTO;
+import com.epam.test_generator.services.TokenService;
+import com.google.gson.Gson;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+@RunWith(MockitoJUnitRunner.class)
+public class LoginControllerTest {
+
+
+    private MockMvc mockMvc;
+
+    @Mock
+    TokenService tokenService;
+
+    @InjectMocks
+    LoginController loginController;
+
+    @Mock
+    LoginUserDTO loginUserDTO;
+
+
+    @Before
+    public void setUp() throws Exception {
+        mockMvc = MockMvcBuilders.standaloneSetup(loginController)
+            .setControllerAdvice(new GlobalExceptionController())
+            .build();
+    }
+
+    @Test
+    public void loginTest_200() throws Exception {
+        LoginUserDTO user = new LoginUserDTO();
+        user.setPassword("test");
+        user.setEmail("test@test.ru");
+        when(tokenService.getToken(loginUserDTO)).thenReturn(anyString());
+        Gson gson = new Gson();
+        String json = gson.toJson(user);
+        mockMvc.perform(post("/login").contentType(MediaType.APPLICATION_JSON).content(json))
+            .andDo(print()).andExpect(status().isOk());
+    }
+
+    @Test
+    public void loginTest_incorrectEmail_400() throws Exception {
+        LoginUserDTO user = new LoginUserDTO();
+        user.setPassword("test");
+        user.setEmail("test");
+        Gson gson = new Gson();
```

Comment:
Please make it a class level property and reuse in methods. Don't create a new object in every method.